Automattic/wp-calypso
# Hide Store tab on Desktop

### Problem
Desktop does not yet have Store on Calypso extension enabled. Until that happens, the "Store" tab on Stats page results in an error.

<img width="400" alt="screen shot 2017-11-13 at 10 12 45 am" src="https://user-images.githubusercontent.com/1922453/32703470-45a54112-c85b-11e7-99ef-1eb8801ff20d.png">

### Fix 
Hide the tab on Desktop environment, but leave it enabled in development, stage, and production.

### Test
1. Navigate Calypso to a Site with jetpack and woocommerce enabled
2. See the "Store" tab rendered

<img width="476" alt="screen shot 2017-11-13 at 9 44 27 am" src="https://user-images.githubusercontent.com/1922453/32703479-78e5c470-c85b-11e7-9d16-03a4ce815acb.png">

3. Fire up `release/3.1.0` branch of wp-desktop
4. Using the same site, see the "Store" tab render

<img width="770" alt="screen shot 2017-11-13 at 9 50 27 am" src="https://user-images.githubusercontent.com/1922453/32703498-b766eaa8-c85b-11e7-8677-d262c8030c68.png">

5. Checkout the matching release of calypso and cherry pick these changes
```
cd calypso
git fetch origin release/desktop/3.1.0
git checkout -b release/desktop/3.1.0
git git cherry-pick 5104f5c1cfae7978b3e0eb803999c28957cefbb7
```
6. See that the "Store" tab does not render. (NOTE: I'm getting an error on `make run`, will update when solved)


cc @justinshreve
